plugin/impl.go

```diff
@@ -84,8 +84,22 @@ func (p *Plugin) Execute() error {
 			return err
 		}
 
+		isEnterprisePRCheck := false
+		if ossPR, ok := p.settings.params["OSS_PULL_REQUEST"]; ok && ossPR != "" && name == "grafana-enterprise" {
+			isEnterprisePRCheck = true
+		}
+
+		// Enterprise PR check shouldn't run on main branch, this means that the PR comes from a fork
+		if isEnterprisePRCheck && branch == "main" {
```

Comment:
For example, in OSS, we have:

```
- image: grafana/drone-downstream
  name: trigger-enterprise-downstream
  settings:
    params:
    - SOURCE_BUILD_NUMBER=${DRONE_COMMIT}
    - SOURCE_COMMIT=${DRONE_COMMIT}
    - OSS_PULL_REQUEST=${DRONE_PULL_REQUEST}
    repositories:
    - grafana/grafana-enterprise@main
    server: https://drone.grafana.net
    token:
      from_secret: drone_token
```
What if we skip the `OSS_PULL_REQUEST` param and check if that flag is not present?

Reply:
It shouldn't change anything as, for main downstream builds, the `DRONE_PULL_REQUEST` env variable will be empty so `isEnterprisePRCheck` will be `false`; but it may be easier to read/understand.

Reply:
I did the change on OSS: https://github.com/grafana/grafana/pull/50681
As a result, I could remove this `ossPR != ""` from line 88 but I think it's safer to keep it, WDYT?